superset/assets/javascripts/dashboard/v2/components/gridComponents/Row.jsx

```diff
@@ -92,6 +93,7 @@ class Row extends React.PureComponent {
       onResizeStop,
       handleComponentDrop,
       editMode,
+      cells,
```

Comment:
@graceguo-supercat  what are cells? (more descriptive name please) 

it looks like they are only needed by row + column? please us `containers/DashboardComponent` to pass only for those components instead of passing them around everywhere.